portal-api/api-tests/v1/facility-dates/cover-end-date.api-test.js

```diff
@@ -64,6 +64,14 @@ describe('hasAllCoverEndDateValues()', () => {
 });
 
 describe('updateCoverEndDate', () => {
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
```

Comment:
For anyone looking back at this PR in future, this change fixes a flakey API test that would sometimes fail because of too much real time elapsing between 2 operations in the test